android/src/main/java/com/facebook/reactnative/androidsdk/RCTLoginButton.java

```diff
@@ -63,7 +61,7 @@ protected void onCurrentAccessTokenChanged(
                     WritableMap event = Arguments.createMap();
                     event.putString("type", "logoutFinished");
                     ReactContext context = (ReactContext) getContext();
-                    context.getJSModule(RCTEventEmitter.class).receiveEvent(
+                    context.getJSModule(ReactEventEmitter.class).receiveEvent(
```

Comment:
Out of curiosity, why not the style that is documented? Using: RCTDeviceEventEmitter?

https://reactnative.dev/docs/native-modules-android#sending-events-to-javascript

Reply:
Didn't see that, I think you are right it should be using `RCTDeviceEventEmitter` instead, I'll change and test it tomorrow. Turning this pr to draft for now until it's ready.

Reply:
After all, this should be `RCTEventEmitter` according to https://reactnative.dev/docs/native-components-android#events, not sure why it's marked deprecated on AndroidStudio, but currently it's a won't fix.